Tests/AppTests/SourceTreeLogic+Tests.swift

```diff
@@ -200,8 +231,8 @@ extension SourceTreeLogicTests {
 
   /// Adds a `request.json` of type `.ok` with a `response.json`.
   /// - Parameters:
-  ///   - url: The `URL` where to create the request file.
-  ///   - addResponse: If `false`, `response.json` won't be created. This will lead to voiding the validity of the request.
+  /// - url: The `URL` where to create the request file.
+  /// - addResponse: If `false`, `response.json` won't be created. This will lead to voiding the validity of the request.
```

Comment:
```suggestion
  ///   - url: The `URL` where to create the request file.
  ///   - addResponse: If `false`, `response.json` won't be created. This will lead to voiding the validity of the request.
```